Islandora/Alpaca
# pass event type as event header to milliner

**GitHub Issue**: (https://github.com/Islandora-CLAW/CLAW/issues/740)

* Other Relevant Links (https://github.com/Islandora-CLAW/islandora/pull/154)

# What does this Pull Request do?
Passes the event type through from the Drupal event object to Milliner's node stream

# What's new?
just mapping event.object.type to a header called Event


# How should this be tested?
* pull down this PR and package up the feature and update in karaf ([how i did this](https://gist.github.com/elizoller/3cf3b2e304c156eab7ff8f4ab674ecda))
* make a drupal node create or update (probably easiest to also have the related PR in above)
* check to make sure the header is present when it gets to milliner 

# Additional Notes:
I kind of made up this header. There are many different ways to do this. It could be an entirely separate stream of events from drupal and/or it could point to a separate milliner endpoint. I'm definitely open to whatever is deemed as the best practice - this was just a starting point to get the pieces connected.

# Interested parties
@Islandora-CLAW/committers
